SPECS/flux/flux.spec

```diff
@@ -113,13 +109,12 @@ RUSTFLAGS=%{rustflags} cargo test --release
 %postun -n libflux -p /sbin/ldconfig
 
 %files -n libflux
-%{_libdir}/libflux.so.%{version}
```

Comment:
Why did we need to change that? Could you provide me some context?

Reply:
Oh yeah, so this pacakge produces file `liblfux.so` which is set as part of I believe systemd requirements inside of `influxdb` package. When trying to install `influxdb` using tdnf given the rpm it says nothing can provide `liblfux.so(64 bit)` file, and when checking what `libflux` package provides it says `libflux.so.0.191-1.cm2` which follows this format, so tdnf does not realize it is the file it is looking for. But I cannot control the require systemd macro sets, so I change the name of the file we provide here to match what tdnf is looking for.

Reply:
Thanks. Per our offline discussion, we've found out how to make influxdb work with versioned .so file, so let's apply those changes.